src/bin/p-s.js

```diff
@@ -73,9 +83,30 @@ function onInit() {
     'Check out your scripts in there. Go ahead and update them and add descriptions to the ones that need it'
   ))
   log.info(colors.gray('Your package.json scripts have also been updated. Run `npm start -- --help` for help'))
+  log.info(colors.gray(
```

Comment:
I thought about that, but decided not to... Maybe I should. Would be slightly easier to maintain.